version/version.go

```diff
@@ -0,0 +1,3 @@
+package version
+
+var Version = "v0.0.0-dev"
```

Comment:
TODO: Remove this file from `.gitignore`.

Reply:
Just checked and I believe it's removed: https://github.com/epinio/epinio/pull/274/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947L18

Reply:
:facepalm: Apologies. I seem to be very out of it this morning. Hopefully not for the entire day. I should know to look at the entire PR changes, not just one commit. Retracting.